python/llm/dev/test/run-example-tests.sh

```diff
@@ -0,0 +1,15 @@
+# LLAMA2 example test
+if [ ! -d $ORIGINAL_LLAMA2_PATH ]; then
```

Comment:
I do not find the place for defining `ORIGINAL_LLAMA2_PATH`, should we put the downloading into the yml file as other tests?

Reply:
> Should we put the downloading into the yml file as other tests?

@hkvision what do you think?

Reply:
I think putting the download together with run script is better since in the future we only need to edit one file?

Reply:
If later we put the example tests directly into the nightly test yml, they may share the downloaded models, so it may be better then to put the downloading into the yml files. But anyway, we could keep the workflow as it for now and change them later. :)

Reply:
`if [ ! -d $ORIGINAL_LLAMA2_PATH ]` there is path check, if the models are downloaded elsewhere, here still won't download again, no?

Reply:
> `if [ ! -d $ORIGINAL_LLAMA2_PATH ]` there is path check, if the models are downloaded elsewhere, here still won't download again, no?

yes, as long as the working directory is the same

Reply:
Yes, the only thing is for other tests, the model may not be downloaded yet before we run into this sh file. Thus for all other tests, the downloading part is put into the yml file before the sh file, so can be safely shared by all tests.

Reply:
Anyway, we could keep as it for now :)